.github/workflows/build_and_test.yml

```diff
@@ -804,7 +804,9 @@ jobs:
 
       - name: Downloading vm-builder
         run: |
-          curl -fL https://github.com/neondatabase/autoscaling/releases/download/$VM_BUILDER_VERSION/vm-builder -o vm-builder
+          # Custom vm-builder since there has not yet been a release
+          # felix-monitor-testing on aws
+          curl -fL ec2-18-217-40-54.us-east-2.compute.amazonaws.com:8000/vm-builder -o vm-builder
```

Comment:
Is the comment relevant?

Reply:
Just explaining the temporary change. See comments below for more context.